cmd/tetragon/main.go

```diff
@@ -440,6 +443,51 @@ func tetragonExecute() error {
 		}
 	}
 
+	// Create a new health server and mark it as serving.
+	// This should be done after the main gRPC server is started.
+	healthServer := gh.NewServer()
+	healthServer.SetServingStatus("liveness", grpc_health_v1.HealthCheckResponse_SERVING)
+
+	// Create a new gRPC server for health checks and register the healthServer.
+	grpcHealthServer := grpc.NewServer()
+	grpc_health_v1.RegisterHealthServer(grpcHealthServer, healthServer)
+
+	// Start the gRPC server for the health checks.
+	go func() {
+		// the gRPC server for the health checks listens on port 6789
+		listener, err := net.Listen("tcp", ":6789")
```

Comment:
probably better to make the port configurable 💭

Reply:
Done. I have also added a way to disable the gRPC server if needed. All of these are now also configurable through configmap.